### PR TITLE
{lang}[GCCcore/9.3.0] Ruby v2.7.2

### PR DESCRIPTION
**Pull request by jedokaplan**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,158 @@
+name = 'Ruby'
+version = '2.7.1'
```
Comment (boegel) [inline easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-9.3.0.eb:2]: @jedokaplan We should go with the latest Ruby version, done in https://github.com/jedokaplan/easybuild-easyconfigs/pull/1

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-9.3.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-9.3.0.eb:2]: @jedokaplan ping change request above?

Comment (boegel) [reply easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-9.3.0.eb]: fixed in e75752f

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3551.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/5631edaeda2092b875ac2e5f6e7715b2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2614.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/1bf3386fb35a7bb7cb4a1b45f3a52360 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/2f2f0ca8c087f7990d1cba95a6207a42 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @jedokaplan!

**Pull request merged**